src/racing/actors.c

```diff
@@ -125,188 +129,188 @@ void actor_init(struct Actor *actor, Vec3f startingPos, Vec3s startingRot, Vec3f
+            actor->unk_08 = 17.0f;
+            break;
+        case 33:
+            actor->flags |= 0x4000;
+            actor->state = 0x0043;
+            actor->boundingBoxSize = 3.0f;
+            actor->unk_08 = 17.0f;
+            break;
+        case 29:
+            actor->flags |= 0x4000;
+            actor->state = 0x0043;
+            actor->boundingBoxSize = 3.0f;
+            actor->unk_08 = 17.0f;
+            break;
+        case 30:
+            actor->flags |= 0x4000;
+            actor->state = 0x0019;
+            actor->boundingBoxSize = 3.0f;
+            actor->unk_08 = 7.0f;
+            break;
+        case 31:
+            actor->flags |= 0x4000;
+            actor->state = 0x0019;
+            actor->boundingBoxSize = 3.0f;
+            actor->unk_08 = 7.0f;
+            break;
+        case 32:
+            actor->flags |= 0x4000;
+            actor->state = 0x0019;
+            actor->boundingBoxSize = 3.0f;
+            actor->unk_08 = 7.0f;
+            break;
+        case ACTOR_PALM_TREE:
+            actor->flags |= 0x4000;
+            actor->state = 0x003C;
+            actor->boundingBoxSize = 3.0f;
+            actor->unk_08 = 13.0f;
+            break;
+        case ACTOR_FAKE_ITEM_BOX:
+            actor->flags = actor->flags | 0x4000 | 0x1000;
+            actor->unk_08 = 0.35f;
+            actor->boundingBoxSize = 1.925f;
+            func_802ADDC8(&actor->unk30, 1.925f, actor->pos[0], actor->pos[1], actor->pos[2]);
+            break;
+        case ACTOR_HOT_AIR_BALLOON_ITEM_BOX:
+            actor->flags |= 0x4000;
+            actor->unk_04 = 0;
+            actor->state = 5;
+            actor->boundingBoxSize = 5.5f;
+            break;
+        case ACTOR_ITEM_BOX:
+            actor->flags |= 0x4000;
+            actor->unk_04 = 0;
+            actor->state = 0;
+            actor->boundingBoxSize = 5.5f;
+            break;
+        case ACTOR_PIRANHA_PLANT:
+            actor->flags |= 0x4000;
+            actor->state = 0x001E;
+            actor->boundingBoxSize = 5.0f;
+            break;
+        default:
+            break;
     }
 }
 
-void func_80297230(Camera *arg0, struct Actor *arg1) {
+void actor_not_render_on_a_camera(Camera *arg0, struct Actor *arg1) {
     switch(arg0 - camera1) {
-        case 0:
-            arg1->flags &= ~1;
+        case PLAYER_ONE:
+            arg1->flags &= ~(1<<PLAYER_ONE);
             break;
-        case 1:
-            arg1->flags &= ~2;
+        case PLAYER_TWO:
+            arg1->flags &= ~(1<<PLAYER_TWO);
             break;
-        case 2:
-            arg1->flags &= ~4;
+        case PLAYER_THREE:
+            arg1->flags &= ~(1<<PLAYER_THREE);
             break;
-        case 3:
-            arg1->flags &= ~8;
+        case PLAYER_FOUR:
+            arg1->flags &= ~(1<<PLAYER_FOUR);
             break;
     }
 }
 
-void func_802972B8(Camera *arg0, struct Actor *arg1) {
+void actor_render_on_a_camera(Camera *arg0, struct Actor *arg1) {
```

Comment:
my main issue is just the _a_ part.

set_actor_rendered_flag ?

Reply:
![image](https://github.com/n64decomp/mk64/assets/59367621/43074cce-04c5-46e6-b783-586c18a06236)
like it's a function complementary to actor_not_rendered, I name it actor_rendered